src/test/java/core/basesyntax/service/ParserServiceImplTest.java

```diff
@@ -0,0 +1,81 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.impl.ParserServiceImpl;
+import core.basesyntax.strategy.handler.BalanceOperationHandler;
+import core.basesyntax.strategy.handler.OperationHandler;
+import core.basesyntax.strategy.handler.PurchaseOperationHandler;
+import core.basesyntax.strategy.handler.ReturnOperationHandler;
+import core.basesyntax.strategy.handler.SupplyOperationHandler;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ParserServiceImplTest {
+    private static List<String> lines;
+    private static Map<Operation, OperationHandler> handlerMap;
+    private static ParserServiceImpl parserService;
+
+    @Before
+    public void setUp() {
+        lines = new ArrayList<>();
+        handlerMap = new HashMap<>();
+        handlerMap.put(Operation.BALANCE, new BalanceOperationHandler());
+        handlerMap.put(Operation.RETURN, new ReturnOperationHandler());
+        handlerMap.put(Operation.SUPPLY, new SupplyOperationHandler());
+        handlerMap.put(Operation.PURCHASE, new PurchaseOperationHandler());
+        parserService = new ParserServiceImpl();
+    }
+
+    @Test
+    public void parse_listString_ok() {
+        lines.add("type,fruit,quantity");
+        lines.add("b,banana,20");
+        lines.add("b,apple,100");
+        lines.add("s,banana,100");
+        lines.add("p,banana,13");
+        lines.add("r,apple,10");
+        lines.add("p,apple,20");
+        lines.add("p,banana,5");
+        lines.add("s,banana,50");
+        Integer expected = 8;
+        Integer actual = parserService.parseLines(lines).size();
+        assertEquals(expected, actual);
```

Comment:
I think that the logic in this test and in parse_resultTransaction_ok is pretty much the same
let's check both size and content in one test, and remove the other